src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -139,7 +690,379 @@ try {
   console.log('Error : ', error);
 }
 ```
+</InlineFilter>
+
+<InlineFilter filters={["swift"]}>
+
+### Monitor upload progress
```

Comment:
I think above I had "Query ongoing transfers". Maybe that should actually be changed to "Monitor ongoing transfers"? How is this different than that other heading in Android?